frontend/packages/topology/locales/zh/topology.json

```diff
@@ -0,0 +1,106 @@
+  "A unique name given to the Application grouping to label your resources.": "为应用程序组指定一个唯一名称以标记您的资源。",
+  "Application": "应用程序",
+  "Select an Application for your grouping or no Application group to not use an Application grouping.": "为您的组选择一个应用程序，或选择没有应用程序组以不使用应用程序分组。",
+  "Application name": "应用程序名称",
+  "all applications": "所有应用程序",
+  "no application group": "无应用程序组",
+  "Error moving connection": "移动连接出错",
+  "Error creating connection": "创建连接出错",
+  "Add resources": "添加资源",
+  "Build {{status}}": "构建 {{status}}",
+  "Monitoring alert": "监控警报",
+  "Create a visual connector": "创建可视连接器",
+  "Edit source code": "编辑源代码",
+  "Open URL": "打开 URI",
+  "Zoom In": "放大",
+  "Zoom Out": "缩小",
+  "Fit to screen": "适合于屏幕",
+  "Reset view": "重置视图",
+  "Layout 1": "布局 1",
+  "Layout": "布局",
+  "Layout 2": "布局 2",
+  "Move": "移动",
+  "Access context menu": "访问上下文菜单",
+  "View details in side panel": "在侧面面板中查看详情",
+  "Access create connector handle": "访问创建连接器句柄",
+  "Open quick search modal": "打开快速搜索模式",
+  "Builds:": "构建：",
+  "CPU": "CPU",
+  "No {{metricLabel}} metrics available.": "没有 {{metricLabel}} 指标可用。",
+  "{{metricLabel}} usage by Pod": "Pod 的 {{metricLabel}} 使用量",
+  "{{ready, number}} of {{count, number}} Pod_0": "{{ready, number}}（共 {{count, number}}） 个 pod",
+  "{{length}} Pods": "{{length}} Pod",
+  "Rollout in progress...": "正在推出......",
+  "Select an Application group to add the component <2>{resource.metadata.name}</2> to": "选择一个应用程序组将组件 <2>{resource.metadata.name}</2> 添加到",
+  "Save": "保存",
+  "Connect <1>{{sourceLabel}}</1> to": "连接 <1>{{sourceLabel}}</1> 到",
+  "Unable to move connector of type {{type}}.": "无法移动类型 {{type}} 的连接器。",
+  "No resources found": "没有找到资源",
+  "To add content to your Project, create an Application, component or service using one of these options.": "要在项目中添加内容，使用以下任一选项创建一个应用程序、组件或服务。",
+  "Topology": "拓扑",
+  "Select a Project to view the topology": "选择一个项目查看拓扑",
+  "Shortcuts": "捷径",
+  "View shortcuts": "查看捷径",
+  "List view": "列表视图",
+  "Topology view": "拓扑视图",
+  "Quick search bar": "快速搜索条",
+  "Add to Project": "添加至项目",
+  "No results": "没有结果",
+  "Quick search button": "快速搜索按钮",
+  "Provided by {{provider}}": "由 {{provider}} 提供",
+  "Quick search list": "快速搜索列表",
+  "View all results for \"{{searchTerm}}\" ({{totalItems, number}})": "查看 \"{{searchTerm}}\" ({{totalItems, number}}) 的所有结果",
+  "Quick search": "快速搜索",
+  "Visual connector": "视觉化连接器",
+  "Binding connector": "绑定连接器",
+  "Traffic connector": "流量连接器",
+  "Resources": "资源",
+  "Connections": "连接",
+  "Kiali link": "Kiali 链接",
+  "Kiali Graph view": "Kiali Graph 视图",
+  "Show groups": "显示组",
+  "Expand groups": "扩展组",
+  "Pod count": "Pod 数量",
+  "Labels": "标签",
+  "Application groupings": "应用程序组",
+  "Connectivity": "连接性",
+  "Consumption": "消耗",
+  "Collapse groups": "折叠组",
+  "Show": "显示",
+  "Display options": "显示选项",
+  "Clear all filters": "清除所有过滤器",
+  "Filter by resource": "按资源过滤",
+  "Find by name...": "按名称查找......",
```

Comment:
Incorrect ellipsis `"Find by name...": "按名称查找..."`

Reply:
@rh-tchuang - Would your translators be able to confirm these ellipses are correct for simplified Chinese? Thanks.

Reply:
@rh-tfu ^

Reply:
@sg00dwin - Closing since the translator confirmed it was as expected.

> rh-tfu 14 hours ago 
Grammatically, the ellipses in simplified Chinese consists of 6 dots, although the ellipses with 3 dots are widely used as well.